adapters/kalao

```diff
@@ -0,0 +1,6 @@
+this.name = "Kalao";
+this.symbol = Kalao;
+this.token = "0x11AC3118309A7215c6d87c7C396e2DF333Ae3A9C";
```

Comment:
Make sure to pass the tests before requesting a review
You can find EVM adapter examples in the main repository